src/utils/constants/index.js

```diff
@@ -82,6 +82,12 @@ const SETTINGS_PRIVACY_URL = 'https://home.helio.social/privacy';
 
 const CONTACT_SCREEN_MAX_SELECTED_USERS = 99;
 
+const ONESIGNAL_PN_TYPE = {
+  PDP: 'pdp',
+  TOPIC: 'topic',
+  PROFILE: 'profile'
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ONESIGNAL_PN_TYPE` constant is a good addition for maintaining the types of OneSignal push notifications in one place. This will improve readability and maintainability of the code by avoiding hard-coded strings throughout the codebase.
```diff
+ const ONESIGNAL_PN_TYPE = {
+   PDP: 'pdp',
+   TOPIC: 'topic',
+   PROFILE: 'profile'
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->